Android/BacktraceAndroidBackgroundUnhandledExceptionHandler.java

```diff
@@ -34,15 +37,19 @@ public BacktraceAndroidBackgroundUnhandledExceptionHandler(String gameObject, St
     }
 
     @Override
-    public void uncaughtException(Thread thread, Throwable exception) {
+    public void uncaughtException(final Thread thread, final Throwable exception) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.CUPCAKE && mRootHandler != null && shouldStop == false) {
-            if(Looper.getMainLooper().getThread().getId() == thread.getId()) {
-                // prevent from sending exception happened to main thread - we will catch them via unity logger
-                return;
+            if (exception instanceof Exception) {
```

Comment:
Why do we need this? Should we not care about Throwable Error?

Reply:
@rqbacktrace @vlussenburg this is a code from backtrace-android because I wanted to be sure we're doing the same thing. I think we can remove it.